src/Satooshi/Bundle/CoverallsV1Bundle/Api/Jobs.php

```diff
@@ -186,7 +186,5 @@ public function getJsonFile()
         if (isset($this->jsonFile)) {
```

Comment:
To be quite honest, I would suggest to remove all of these conditions and just return the value, regardless of whether it satisfies `isset()` or not - it's the same after all.